smarts/core/sensors.py

```diff
@@ -901,11 +900,11 @@ def __init__(self, vehicle, sim, plan, horizon=32):
         self._plan = plan
         self._horizon = horizon
 
-    def __call__(self):
+    def __call__(self) -> RoadWaypoints:
         veh_pt = self._vehicle.pose.point
         lane = self._road_map.nearest_lane(veh_pt)
         if not lane:
-            return RoadWaypoints(lanes={}, route_waypoints=[])
+            return RoadWaypoints(lanes={})
```

Comment:
Since this affects the interface (and so is a "breaking change"), could you please add a line about this in CHANGELOG.md?

Reply:
I have added the change into changelog.md.